content/docs/csidriver/features/powerflex.md

```diff
@@ -925,6 +937,25 @@ externalAccess: "10.0.0.0/24"
 
 This means that we allow for NFS Export created by driver to be consumed by address range `10.0.0.0-10.0.0.255`.
 
+## Configuring NFS independent of SDC
+
+Starting from CSM 1.12.0, the CSI PowerFlex driver supports configuring NFS independent of SDC. This separation is helpful in scenarios where an SDC is not available in the cluster or additional network interfaces do not need to be deployed.
+
+To enable this feature you need to disable the automatic `sdc` deployment from the values file and provide the interface names mapping for each of the nodes that are being used.
```

Comment:
Rephrase as "To disable SDC deployment, update values file and provide the interface names mapping....". Reason, I dont want to convey the message that disabling SDC is a feature.